### PR TITLE
Start cli module.

### PR DESCRIPTION
**Pull request by smarlowucf**
Add delete and upload image tarball commands.

### CODE REVIEW

**Review by jgleissner — APPROVED** [review]

File: aliyun_img_utils/aliyun_utils.py [hunk 1]
```diff
@@ -20,10 +20,154 @@
...
+            "{}: {}".format(type(error).__name__, error),
+            no_color,
+            fg='red'
+        )
+        sys.exit(1)
+
+
+def click_progress_callback(offset, total_size):
+    """
+    Update the module level progress bar with image download progress.
```
Comment (rjschwei) [inline aliyun_img_utils/aliyun_utils.py:124]: s/download/upload/ ?

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: aliyun_img_utils/aliyun_cli.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+)
+@add_options(shared_options)
+@click.pass_context
+def delete(context, blob_name, **kwargs):
+    process_shared_options(context.obj, kwargs)
+    config_data = get_config(context.obj)
+    logger = get_logger(config_data.log_level)
+
+    with handle_errors(config_data.log_level, config_data.no_color):
+        uploader = AliyunImage(
```
[the remark this answers is not in the document]
Comment (smarlowucf) [reply aliyun_img_utils/aliyun_cli.py:237]: image shadows another variable, went with `aliyun_image`.

File: aliyun_img_utils/aliyun_cli.py [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+            log_level=config_data.log_level,
+            log_callback=logger
+        )
+
+        keyword_args = {'blob_name': blob_name}
+
+        if config_data.log_level != logging.ERROR:
+            keyword_args['progress_callback'] = click_progress_callback
+
+        deleted = uploader.delete_image_tarball(blob_name, **keyword_args)
```
[the remark this answers is not in the document]
Comment (smarlowucf) [reply aliyun_img_utils/aliyun_cli.py:251]: Right, this is my intention but I have no framework image functions yet. So my plan was to update the CLI function similar to the suggestion once we have a delete_image function.

**Comment by smarlowucf** [comment]
@rjschwei updated per comments. The delete function I plan to change similar to your suggestions once we have the image level functions in place.

**Review by rjschwei — APPROVED** [review]

**Pull request merged**